docs/tutorials/image_ops.ipynb

```diff
@@ -1,535 +1,665 @@
 {
```

Comment:
<p>Hmmmm, not sure why the quiet flag didn't supress this but could we remove the downloading outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Sure. Done.